searx/utils.py

```diff
@@ -615,6 +615,48 @@ def _get_fasttext_model() -> "fasttext.FastText._FastText":  # type: ignore
     return _FASTTEXT_MODEL
 
 
+def _get_iframe_src(url):
+    parsed_url = urlparse(url)
+    iframe_src = None
```

Comment:
I think the function should be renamed to something similar to ``def get_embeded_stream_url(url)`` .. and a doc-string is issing that describes what distinguishes the input URL from the returned one .. what streamning services supported (or what types of URLs).